packages/spotlight/src/spotlight.js

```diff
@@ -253,12 +254,16 @@ const Spotlight = (function () {
 		let next;
 
 		if (lastContainerId) {
+			const position = getLastPointerPosition();
+
 			// walk up the chain of containers from the last to attempt to find a target
 			next = getContainersForNode(getContainerNode(lastContainerId)).reverse();
 
 			// only prepend last focused if it exists so that Spotlight.focus() doesn't receive
 			// a falsy target
-			const lastFocused = getContainerLastFocusedElement(lastContainerId);
+			const lastFocused = getContainerConfig(lastContainerId).overflow && getNearestTargetFromPosition(position, lastContainerId) ||
```

Comment:
This logic favors the "nearest" over the "last focused" independent of the visibility of the last focused. I suspect we'll get a report on this at some point but I think that is more edge case than the reported bug and fix so happy to move ahead with this.